src/en/authors-charm-metadata.md

```diff
@@ -104,5 +104,43 @@ and whether it's designed for deployment as a
   - if the charm is subordinate, it must contain at least one `requires`
     relation with container scope.
 
+`resources` defines ???
+
+```yaml
+resources:
+  example:
+    type: file # "file" is the only type supported currently
+    filename: example.tar.gz
+    description: example resource
+```
+
+`payloads` defines ???
+
+```yaml
+payloads:
+  example:
+    type: kvm
+```
+
+`extra-bindings` represents an extra bindable endpoint that is not a relation.
+
+```yaml
+extra-bindings:
+  endpoint1:
+  endpoint2:
+```
+
+Endpoint names are strings and must not match existing relation names from
+the Provides, Requires, or Peers metadata sections. The values beside each
+endpoint name must be left out (i.e. "foo": <anything> is invalid).
+
+Other available fields are:
+
+  - `format` ???
+  - `categories` ???
```

Comment:
`format` is for historical reasons and is not used anymore. `categories` have since become `tags`